command: /usr/bin/python /u/pedronis/PyPy/dist/pypy/tool/alarm.py 900 /u/pedronis/PyPy/dist/pypy/interpreter/py.py --oldstyle /u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_unary.py
run by: pedronis@ratthing-b246
sys.platform: linux2
sys.version_info: (2, 3, 5, 'final', 0)
startdate: Thu Apr 28 15:09:30 2005
pypy-revision: 11549
============================================================
faking <type 'unicode'>
faking <type 'module'>
faking <type 'file'>
faking <type 'posix.stat_result'>
faking <type 'posix.statvfs_result'>
test_bad_types (__main__.UnaryOpTestCase) ... ok
test_invert (__main__.UnaryOpTestCase) ... ok
test_negation_of_exponentiation (__main__.UnaryOpTestCase) ... ok
test_negative (__main__.UnaryOpTestCase) ... ok
test_no_overflow (__main__.UnaryOpTestCase) ... ok
test_positive (__main__.UnaryOpTestCase) ... ok

----------------------------------------------------------------------
Ran 6 tests in 2.198s

OK
========================== closed ==========================
execution time: 9.84963107109 seconds
exit status: 0
